docs/content/快速上手/mysql-postgres-tutorial-zh.md

```diff
@@ -0,0 +1,317 @@
+           (default,"car battery","12V car battery"),
+           (default,"12-pack drill bits","12-pack of drill bits with sizes ranging from #40 to #3"),
+           (default,"hammer","12oz carpenter's hammer"),
+           (default,"hammer","14oz carpenter's hammer"),
+           (default,"hammer","16oz carpenter's hammer"),
+           (default,"rocks","box of assorted rocks"),
+           (default,"jacket","water resistent black wind breaker"),
+           (default,"spare tire","24 inch spare tire");
+    
+    CREATE TABLE orders (
+      order_id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      order_date DATETIME NOT NULL,
+      customer_name VARCHAR(255) NOT NULL,
+      price DECIMAL(10, 5) NOT NULL,
+      product_id INTEGER NOT NULL,
+      order_status BOOLEAN NOT NULL -- Whether order has been placed
+    ) AUTO_INCREMENT = 10001;
+    
+    INSERT INTO orders
+    VALUES (default, '2020-07-30 10:08:22', 'Jark', 50.50, 102, false),
+           (default, '2020-07-30 10:11:09', 'Sally', 15.00, 105, false),
+           (default, '2020-07-30 12:00:30', 'Edward', 25.25, 106, false);
+    ```
+#### 在 Postgres 数据库中准备数据
+1. 进入 Postgres 容器
+    ```shell
+    docker-compose exec postgres psql -h localhost -U postgres
+    ```
+2. 创建表 `shipments`，并插入数据
+    ```sql
+   -- PG
+   CREATE TABLE shipments (
+      shipment_id SERIAL NOT NULL PRIMARY KEY,
+      order_id SERIAL NOT NULL,
+      origin VARCHAR(255) NOT NULL,
+      destination VARCHAR(255) NOT NULL,
+      is_arrived BOOLEAN NOT NULL
+    );
+    ALTER SEQUENCE public.shipments_shipment_id_seq RESTART WITH 1001;
+    ALTER TABLE public.shipments REPLICA IDENTITY FULL;
+    INSERT INTO shipments
+    VALUES (default,10001,'Beijing','Shanghai',false),
+           (default,10002,'Hangzhou','Shanghai',false),
+           (default,10003,'Shanghai','Hangzhou',false);
+    ```
+   
+## 启动 Flink 集群和 Flink SQL CLI
+
+1. 使用下面的命令跳转至 Flink 目录下
+    ```
+    cd flink-1.13.2
+    ```
+   
+2. 使用下面的命令启动 Flink 集群
+    ```shell
+    ./bin/start-cluster.sh
+    ```
+   启动成功的话，可以在 [http://localhost:8081/](http://localhost:8081/) 访问到 Flink Web UI，如下所示：
+
+   ![Flink UI](/_static/fig/mysql-postgress-tutorial/flink-ui.png "Flink UI")
+
+3. 使用下面的命令启动 Flink SQL CLI
+    ```shell
+    ./bin/sql-client.sh
+    ```
+   启动成功后，可以看到如下的页面：
+
+   ![Flink SQL_Client](/_static/fig/mysql-postgress-tutorial/flink-sql-client.png "Flink SQL Client")
+
+## 在 Flink SQL CLI 中使用 Flink DDL 创建表
+首先，开启 checkpoint，每隔3秒做一次 checkpoint
+```sql
+-- Flink SQL                   
+Flink SQL> SET execution.checkpointing.interval = 3s;
+```
+
+然后, 对于数据库中的表 `products`, `orders`, `shipments`， 使用 Flink SQL CLI 创建对应的表，用于同步这些底层数据库表的数据
+```sql
+-- Flink SQL
+Flink SQL> CREATE TABLE products (
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+  ) WITH (
+    'connector' = 'mysql-cdc',
+    'hostname' = 'localhost',
+    'port' = '3306',
+    'username' = 'root',
+    'password' = '123456',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)